container/karaf/src/main/filtered-resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -54,8 +53,8 @@ log4j2.logger.spifly.name = org.apache.aries.spifly
 log4j2.logger.spifly.level = WARN
 
 # Security audit logger
-log4j2.logger.audit.name = org.apache.karaf.jaas.modules.audit
-log4j2.logger.audit.level = INFO
+log4j2.logger.audit.name = audit
+log4j2.logger.audit.level = TRACE
```

Comment:
Did this actually change to TRACE in the default?  Will that blow up logging, or have you noticed changes?